freedomofpress/securedrop
# Remove journalist assignment

## Status

Ready for review

## Description of Changes

Unfortunately, since database migrations support (#1419) did not make it into the feature set for 0.4, we can't ship the journalist assignment feature introduced in #1491 to prod instances. This PR removes this feature. 

I've added to the checklist in #1658 so that we can bring it in later in the 0.4.x release series when database migrations is implemented.

## Testing

How should the reviewer test this PR?

1. Provision development VM
2. Run unit/functional tests
3. Submit a document to the source interface
4. Make admin account, sign into journalist interface
5. Verify journalist assignment box is gone

## Screenshot

Journalist assignment field is gone:

![screen shot 2017-04-26 at 8 31 32 am](https://cloud.githubusercontent.com/assets/7832803/25442955/50e3db18-2a5b-11e7-8747-4388daa48cd6.png)

## Checklist

### If you made changes to the app code:
